pkgs/README.md

```diff
@@ -30,12 +30,23 @@ Before adding a new package, please consider the following questions:
 
 * Is the package ready for general use? We don't want to include projects that are too immature or are going to be abandoned immediately. In case of doubt, check with upstream.
 * Does the project have a clear license statement? Remember that software is unfree by default (all rights reserved), and merely providing access to the source code does not imply its redistribution. In case of doubt, ask upstream.
-* How realistic is it that it will be used by other people? It's good that nixpkgs caters to various niches, but if it's a niche of 5 people it's probably too small.
+* How realistic is it that it will be used by other people? It's good that nixpkgs caters to various niches, but if it's a niche of 5 people it's probably too small. A good estimate is checking upstream issues and pull requests, or other software repositories.
+* Is the software maintained upstream? Especially security-critical components should see regular maintenance.
 * Are you willing to maintain the package? You should care enough about the package to be willing to keep it up and running for at least one complete Nixpkgs' release life-cycle.
   * In case you are not able to maintain the package you wrote, you can seek someone to fill that role, effectively adopting the package.
 
 If any of these questions' answer is no, then you should probably not add the package.
 
+Special care has to be taken with security-critical software components.
+* Any package that immediately would need to be tagged with `meta.knownVulnerabilities` is not fit for nixpkgs
```

Comment:
```suggestion
* Any package that immediately would need to be tagged with `meta.knownVulnerabilities` is unlikely to be fit for nixpkgs
```

We should still mark `knownVulnerabilities` that only concern exotic use cases as long as we cannot easily auto-block those usecases. For example, `curl-impersonate` is basically only useful for HTTPS fetching from Cloudflare, so unpatched e.g. SMB issues are not relevant to the real usecase, but ideally should also be marked, but do not mean the package is not useful to have while those issues are technically there.

Reply:
Fair! This is the feedback i am looking for. Olm comes to mind too, with the known vuln yet being commonly used